qutebrowser/mainwindow/prompt.py

```diff
@@ -467,6 +467,27 @@ def prompt_yank(self, sel=False):
         utils.set_clipboard(question.url, sel)
         message.info("Yanked to {}: {}".format(target, question.url))
 
+    @cmdutils.register(
+        instance='prompt-container', scope='window',
+        modes=[usertypes.KeyMode.prompt])
+    def prompt_external_picker(self):
+        """Choose a location using a configured external picker."""
+        picker = config.val.fileselect.folder.command
+        if not picker:
+            log.webview.warning(
+                "No external location picker (fileselect.folder.command) configured."
+            )
+            return
+        folders = shared.choose_file(shared.FileSelectionMode.folder)
+        if not folders:
+            message.info("No folder chosen.")
+            return
+        assert self._prompt is not None
+        question = self._prompt.question
+        # choose_file already checks that this is max one folder
```

Comment:
Maybe add an assert for it here?

Reply:
Done in 65c3007fc07f708e716c9cb14fefd10bf5f240d7